common/src/main/java/us/ilite/common/lib/util/Conversions.java

```diff
@@ -32,6 +32,10 @@ public static double ticksToInches(int ticks) {
         return ticksToRotations(ticks) * SystemSettings.kDriveWheelCircumference;
     }
 
+    public static int inchesToTicks(double inches) {
```

Comment:
Seems like a very generic name for a very specific conversion, or is this conversion applicable for more than the drive train?